### PR TITLE
Remove Degree Symbol in Filenames

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
**Issue:** #4637
**Alexa rank:** n/a

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Removes the `°` symbol when normalising filenames.

### CODE REVIEW

**Review by adamrusted — COMMENTED** [review]
LGTM! Will wait for another approval before merging though.

**Review by service-paradis — APPROVED** [review]
LGTM, thank you @PeterShaggyNoble 🥳

**Pull request merged**
